src/rdb/rdb_raft.c

```diff
@@ -1450,6 +1450,7 @@ rdb_compactd(void *arg)
 				": %d\n", DP_DB(db), base, rc);
 			break;
 		}
+		dss_gc_run(256);
```

Comment:
GC_CREDS ?  Obviously, that would require moving to header...any reason for for the number?

Reply:
moved